Pulsar manager start failed when enable bkvm

```

2020-09-09 19:44:12.036  INFO 48758 --- [ost-startStop-1] .0.jar_punit_nonJtaDataSource=1983259277 : EclipseLink, version: Eclipse Persistence Services - 2.7.5.v20191016-ea124dd158
2020-09-09 19:44:12.037  INFO 48758 --- [ost-startStop-1] herddb.jdbc.BasicHerdDBDataSource        : Booting HerdDB Client, url:jdbc:herddb:localhost:7000?server.mode=standalone&server.start=false, properties:{password=---} clientConfig ClientConfiguration{{password=---}}
2020-09-09 19:44:12.059  INFO 48758 --- [ost-startStop-1] o.e.p.s./.0.0.j.connection               : Not able to detect platform for vendor name [HerdDB[0.19.0, 0]]. Defaulting to [org.eclipse.persistence.platform.database.DatabasePlatform]. The database dialect used may not match with the database you are using. Please explicitly provide a platform using property "eclipselink.target-database".
2020-09-09 19:44:12.067  INFO 48758 --- [ost-startStop-1] o.e.p.s./.0.0.j.connection               : connecting(DatabaseLogin(
	platform=>DatabasePlatform
	user name=> ""
	connector=>JNDIConnector datasource name=>null
))
2020-09-09 19:44:12.068  INFO 48758 --- [ost-startStop-1] o.e.p.s./.0.0.j.connection               : Connected:
	User:
	Database: HerdDB  Version: 0.19.0
	Driver: HerdDB  Version: 0.19.0
2020-09-09 19:44:12.068  INFO 48758 --- [ost-startStop-1] o.e.p.s./.0.0.j.connection               : connecting(DatabaseLogin(
	platform=>DatabasePlatform
	user name=> ""
	connector=>JNDIConnector datasource name=>null
))
2020-09-09 19:44:12.069  INFO 48758 --- [ost-startStop-1] o.e.p.s./.0.0.j.connection               : Connected:
	User:
	Database: HerdDB  Version: 0.19.0
	Driver: HerdDB  Version: 0.19.0
2020-09-09 19:44:12.110  INFO 48758 --- [ost-startStop-1] o.e.p.s./.0.0.j.connection               : /file:/Users/tuteng/streamnative/release/pulsar-manager/build/distributions/pulsar-manager/lib/bkvm-2.0.0-classes.jar_punit_nonJtaDataSource=1983259277 login successful
2020-09-09 19:44:12.137  INFO 48758 --- [ost-startStop-1] herddb.server.ServerSideConnectionPeer   : client jvm-local connected as sa
2020-09-09 19:44:12.608  INFO 48758 --- [ost-startStop-1] herddb.sql.CalcitePlanner                : Error while parsing 'SELECT 1 FROM LEDGER'

org.apache.calcite.tools.ValidationException: org.apache.calcite.runtime.CalciteContextException: From line 1, column 15 to line 1, column 20: Object 'LEDGER' not found
	at org.apache.calcite.prepare.PlannerImpl.validate(PlannerImpl.java:217) ~[calcite-core-1.24.0.jar:1.24.0]
	at herddb.sql.CalcitePlanner.runPlanner(CalcitePlanner.java:565) ~[herddb-core-0.19.0.jar:na]
	at herddb.sql.CalcitePlanner.translate(CalcitePlanner.java:331) ~[herddb-core-0.19.0.jar:na]
	at herddb.server.ServerSideConnectionPeer.handleOpenScanner(ServerSideConnectionPeer.java:447) [herddb-core-0.19.0.jar:na]
	at herddb.server.ServerSideConnectionPeer.requestReceived(ServerSideConnectionPeer.java:183) [herddb-core-0.19.0.jar:na]
	at herddb.network.netty.AbstractChannel.processRequest(AbstractChannel.java:96) [herddb-net-0.19.0.jar:na]
	at herddb.network.netty.AbstractChannel.directProcessPdu(AbstractChannel.java:88) [herddb-net-0.19.0.jar:na]
	at herddb.network.netty.LocalVMChannel.sendOneWayMessage(LocalVMChannel.java:63) [herddb-net-0.19.0.jar:na]
	at herddb.network.netty.AbstractChannel.sendRequestWithAsyncReply(AbstractChannel.java:177) [herddb-net-0.19.0.jar:na]
	at herddb.network.Channel.sendMessageWithPduReply(Channel.java:81) [herddb-net-0.19.0.jar:na]
	at herddb.client.RoutedClientSideConnection.executeScan(RoutedClientSideConnection.java:715) [herddb-core-0.19.0.jar:na]
	at herddb.client.HDBConnection.executeScan(HDBConnection.java:345) [herddb-core-0.19.0.jar:na]
	at herddb.jdbc.HerdDBPreparedStatement.executeQuery(HerdDBPreparedStatement.java:81) [herddb-jdbc-0.19.0.jar:na]
	at org.eclipse.persistence.internal.databaseaccess.DatabaseAccessor.executeSelect(DatabaseAccessor.java:1015) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.internal.databaseaccess.DatabaseAccessor.basicExecuteCall(DatabaseAccessor.java:651) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.internal.databaseaccess.DatabaseAccessor.executeCall(DatabaseAccessor.java:567) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.internal.sessions.AbstractSession.basicExecuteCall(AbstractSession.java:2099) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.sessions.server.ServerSession.executeCall(ServerSession.java:603) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.internal.queries.DatasourceCallQueryMechanism.executeCall(DatasourceCallQueryMechanism.java:275) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.internal.queries.DatasourceCallQueryMechanism.executeCall(DatasourceCallQueryMechanism.java:261) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.internal.queries.DatasourceCallQueryMechanism.executeSelectCall(DatasourceCallQueryMechanism.java:332) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.internal.queries.DatasourceCallQueryMechanism.executeSelect(DatasourceCallQueryMechanism.java:314) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.queries.DataReadQuery.executeNonCursor(DataReadQuery.java:199) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.queries.DataReadQuery.executeDatabaseQuery(DataReadQuery.java:154) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.queries.DatabaseQuery.execute(DatabaseQuery.java:911) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.queries.DataReadQuery.execute(DataReadQuery.java:139) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.internal.sessions.AbstractSession.internalExecuteQuery(AbstractSession.java:3356) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.internal.sessions.AbstractSession.executeQuery(AbstractSession.java:1898) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.internal.sessions.AbstractSession.executeQuery(AbstractSession.java:1880) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.internal.sessions.AbstractSession.executeQuery(AbstractSession.java:1830) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.internal.databaseaccess.DatabasePlatform.checkTableExists(DatabasePlatform.java:3775) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.tools.schemaframework.SchemaManager.checkTableExists(SchemaManager.java:467) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.tools.schemaframework.SchemaManager.checkTableExists(SchemaManager.java:480) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.tools.schemaframework.TableCreator.createTables(TableCreator.java:168) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.tools.schemaframework.TableCreator.createTables(TableCreator.java:150) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.tools.schemaframework.TableCreator.createTables(TableCreator.java:142) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.tools.schemaframework.SchemaManager.createDefaultTables(SchemaManager.java:1030) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryProvider.generateDefaultTables(EntityManagerFactoryProvider.java:110) [org.eclipse.persistence.jpa-2.7.5.jar:na]
	at org.eclipse.persistence.internal.jpa.EntityManagerSetupImpl.writeDDLToDatabase(EntityManagerSetupImpl.java:4436) [org.eclipse.persistence.jpa-2.7.5.jar:na]
	at org.eclipse.persistence.internal.jpa.EntityManagerSetupImpl.writeMetadataDDLToDatabase(EntityManagerSetupImpl.java:4503) [org.eclipse.persistence.jpa-2.7.5.jar:na]
	at org.eclipse.persistence.internal.jpa.EntityManagerSetupImpl.writeDDL(EntityManagerSetupImpl.java:4402) [org.eclipse.persistence.jpa-2.7.5.jar:na]
	at org.eclipse.persistence.internal.jpa.EntityManagerSetupImpl.writeDDL(EntityManagerSetupImpl.java:4275) [org.eclipse.persistence.jpa-2.7.5.jar:na]
	at org.eclipse.persistence.internal.jpa.EntityManagerSetupImpl.deploy(EntityManagerSetupImpl.java:819) [org.eclipse.persistence.jpa-2.7.5.jar:na]
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryDelegate.getAbstractSession(EntityManagerFactoryDelegate.java:222) [org.eclipse.persistence.jpa-2.7.5.jar:na]
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryDelegate.getDatabaseSession(EntityManagerFactoryDelegate.java:200) [org.eclipse.persistence.jpa-2.7.5.jar:na]
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryImpl.getDatabaseSession(EntityManagerFactoryImpl.java:542) [org.eclipse.persistence.jpa-2.7.5.jar:na]
	at org.eclipse.persistence.jpa.PersistenceProvider.createEntityManagerFactoryImpl(PersistenceProvider.java:153) [org.eclipse.persistence.jpa-2.7.5.jar:na]
	at org.eclipse.persistence.jpa.PersistenceProvider.createEntityManagerFactory(PersistenceProvider.java:191) [org.eclipse.persistence.jpa-2.7.5.jar:na]
	at javax.persistence.Persistence.createEntityManagerFactory(Persistence.java:80) [jakarta.persistence-2.2.3.jar:2.2.3]
	at org.bkvm.cache.MetadataCache.<init>(MetadataCache.java:46) [bkvm-2.0.0-classes.jar:na]
	at org.bkvm.api.listeners.ContextInitializer.contextInitialized(ContextInitializer.java:72) [bkvm-2.0.0-classes.jar:na]
	at org.apache.catalina.core.StandardContext.listenerStart(StandardContext.java:4792) [tomcat-embed-core-8.5.31.jar:8.5.31]
	at org.apache.catalina.core.StandardContext.startInternal(StandardContext.java:5256) [tomcat-embed-core-8.5.31.jar:8.5.31]
	at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:150) [tomcat-embed-core-8.5.31.jar:8.5.31]
	at org.apache.catalina.core.ContainerBase$StartChild.call(ContainerBase.java:1421) [tomcat-embed-core-8.5.31.jar:8.5.31]
	at org.apache.catalina.core.ContainerBase$StartChild.call(ContainerBase.java:1411) [tomcat-embed-core-8.5.31.jar:8.5.31]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_201]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [na:1.8.0_201]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [na:1.8.0_201]
	at java.lang.Thread.run(Thread.java:748) [na:1.8.0_201]
Caused by: org.apache.calcite.runtime.CalciteContextException: From line 1, column 15 to line 1, column 20: Object 'LEDGER' not found
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[na:1.8.0_201]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[na:1.8.0_201]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.8.0_201]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423) ~[na:1.8.0_201]
	at org.apache.calcite.runtime.Resources$ExInstWithCause.ex(Resources.java:457) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.SqlUtil.newContextException(SqlUtil.java:857) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.SqlUtil.newContextException(SqlUtil.java:842) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.SqlValidatorImpl.newValidationError(SqlValidatorImpl.java:5009) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.IdentifierNamespace.resolveImpl(IdentifierNamespace.java:179) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.IdentifierNamespace.validateImpl(IdentifierNamespace.java:184) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.AbstractNamespace.validate(AbstractNamespace.java:84) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.SqlValidatorImpl.validateNamespace(SqlValidatorImpl.java:1089) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.SqlValidatorImpl.validateQuery(SqlValidatorImpl.java:1063) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.SqlValidatorImpl.validateFrom(SqlValidatorImpl.java:3191) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.SqlValidatorImpl.validateFrom(SqlValidatorImpl.java:3173) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.SqlValidatorImpl.validateSelect(SqlValidatorImpl.java:3447) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.SelectNamespace.validateImpl(SelectNamespace.java:60) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.AbstractNamespace.validate(AbstractNamespace.java:84) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.SqlValidatorImpl.validateNamespace(SqlValidatorImpl.java:1089) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.SqlValidatorImpl.validateQuery(SqlValidatorImpl.java:1063) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.SqlSelect.validate(SqlSelect.java:232) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.SqlValidatorImpl.validateScopedExpression(SqlValidatorImpl.java:1038) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.SqlValidatorImpl.validate(SqlValidatorImpl.java:746) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.prepare.PlannerImpl.validate(PlannerImpl.java:215) ~[calcite-core-1.24.0.jar:1.24.0]
	... 59 common frames omitted
Caused by: org.apache.calcite.sql.validate.SqlValidatorException: Object 'LEDGER' not found
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[na:1.8.0_201]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[na:1.8.0_201]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.8.0_201]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423) ~[na:1.8.0_201]
	at org.apache.calcite.runtime.Resources$ExInstWithCause.ex(Resources.java:457) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.runtime.Resources$ExInst.ex(Resources.java:550) ~[calcite-core-1.24.0.jar:1.24.0]
	... 78 common frames omitted
```

ping @eolivelli PTAL

The dump file seems to be too big
```
2020-09-09 21:39:42.090  INFO 51244 --- [ost-startStop-1] o.e.p.s./.0.0.j.connection               : Not able to detect platform for vendor name [HerdDB[0.19.0, 0]]. Defaulting to [org.eclipse.persistence.platform.database.DatabasePlatform]. The database dialect used may not match with the database you are using. Please explicitly provide a platform using property "eclipselink.target-database".
2020-09-09 21:39:42.097  INFO 51244 --- [ost-startStop-1] o.e.p.s./.0.0.j.connection               : connecting(DatabaseLogin(
	platform=>DatabasePlatform
	user name=> ""
	connector=>JNDIConnector datasource name=>null
))
2020-09-09 21:39:42.098  INFO 51244 --- [ost-startStop-1] o.e.p.s./.0.0.j.connection               : Connected:
	User:
	Database: HerdDB  Version: 0.19.0
	Driver: HerdDB  Version: 0.19.0
2020-09-09 21:39:42.099  INFO 51244 --- [ost-startStop-1] o.e.p.s./.0.0.j.connection               : connecting(DatabaseLogin(
	platform=>DatabasePlatform
	user name=> ""
	connector=>JNDIConnector datasource name=>null
))
2020-09-09 21:39:42.099  INFO 51244 --- [ost-startStop-1] o.e.p.s./.0.0.j.connection               : Connected:
	User:
	Database: HerdDB  Version: 0.19.0
	Driver: HerdDB  Version: 0.19.0
2020-09-09 21:39:42.142  INFO 51244 --- [ost-startStop-1] o.e.p.s./.0.0.j.connection               : /file:/Users/tuteng/streamnative/release/pulsar-manager/build/distributions/pulsar-manager/lib/bkvm-2.0.0-classes.jar_punit_nonJtaDataSource=1244644352 login successful
2020-09-09 21:39:42.169  INFO 51244 --- [ost-startStop-1] herddb.server.ServerSideConnectionPeer   : client jvm-local connected as sa
2020-09-09 21:39:42.624  INFO 51244 --- [ost-startStop-1] herddb.sql.CalcitePlanner                : Error while parsing 'SELECT 1 FROM LEDGER'

org.apache.calcite.tools.ValidationException: org.apache.calcite.runtime.CalciteContextException: From line 1, column 15 to line 1, column 20: Object 'LEDGER' not found
	at org.apache.calcite.prepare.PlannerImpl.validate(PlannerImpl.java:217) ~[calcite-core-1.24.0.jar:1.24.0]
	at herddb.sql.CalcitePlanner.runPlanner(CalcitePlanner.java:565) ~[herddb-core-0.19.0.jar:na]
	at herddb.sql.CalcitePlanner.translate(CalcitePlanner.java:331) ~[herddb-core-0.19.0.jar:na]
	at herddb.server.ServerSideConnectionPeer.handleOpenScanner(ServerSideConnectionPeer.java:447) [herddb-core-0.19.0.jar:na]
	at herddb.server.ServerSideConnectionPeer.requestReceived(ServerSideConnectionPeer.java:183) [herddb-core-0.19.0.jar:na]
	at herddb.network.netty.AbstractChannel.processRequest(AbstractChannel.java:96) [herddb-net-0.19.0.jar:na]
	at herddb.network.netty.AbstractChannel.directProcessPdu(AbstractChannel.java:88) [herddb-net-0.19.0.jar:na]
	at herddb.network.netty.LocalVMChannel.sendOneWayMessage(LocalVMChannel.java:63) [herddb-net-0.19.0.jar:na]
	at herddb.network.netty.AbstractChannel.sendRequestWithAsyncReply(AbstractChannel.java:177) [herddb-net-0.19.0.jar:na]
	at herddb.network.Channel.sendMessageWithPduReply(Channel.java:81) [herddb-net-0.19.0.jar:na]
	at herddb.client.RoutedClientSideConnection.executeScan(RoutedClientSideConnection.java:715) [herddb-core-0.19.0.jar:na]
	at herddb.client.HDBConnection.executeScan(HDBConnection.java:345) [herddb-core-0.19.0.jar:na]
	at herddb.jdbc.HerdDBPreparedStatement.executeQuery(HerdDBPreparedStatement.java:81) [herddb-jdbc-0.19.0.jar:na]
	at org.eclipse.persistence.internal.databaseaccess.DatabaseAccessor.executeSelect(DatabaseAccessor.java:1015) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.internal.databaseaccess.DatabaseAccessor.basicExecuteCall(DatabaseAccessor.java:651) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.internal.databaseaccess.DatabaseAccessor.executeCall(DatabaseAccessor.java:567) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.internal.sessions.AbstractSession.basicExecuteCall(AbstractSession.java:2099) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.sessions.server.ServerSession.executeCall(ServerSession.java:603) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.internal.queries.DatasourceCallQueryMechanism.executeCall(DatasourceCallQueryMechanism.java:275) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.internal.queries.DatasourceCallQueryMechanism.executeCall(DatasourceCallQueryMechanism.java:261) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.internal.queries.DatasourceCallQueryMechanism.executeSelectCall(DatasourceCallQueryMechanism.java:332) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.internal.queries.DatasourceCallQueryMechanism.executeSelect(DatasourceCallQueryMechanism.java:314) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.queries.DataReadQuery.executeNonCursor(DataReadQuery.java:199) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.queries.DataReadQuery.executeDatabaseQuery(DataReadQuery.java:154) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.queries.DatabaseQuery.execute(DatabaseQuery.java:911) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.queries.DataReadQuery.execute(DataReadQuery.java:139) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.internal.sessions.AbstractSession.internalExecuteQuery(AbstractSession.java:3356) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.internal.sessions.AbstractSession.executeQuery(AbstractSession.java:1898) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.internal.sessions.AbstractSession.executeQuery(AbstractSession.java:1880) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.internal.sessions.AbstractSession.executeQuery(AbstractSession.java:1830) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.internal.databaseaccess.DatabasePlatform.checkTableExists(DatabasePlatform.java:3775) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.tools.schemaframework.SchemaManager.checkTableExists(SchemaManager.java:467) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.tools.schemaframework.SchemaManager.checkTableExists(SchemaManager.java:480) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.tools.schemaframework.TableCreator.createTables(TableCreator.java:168) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.tools.schemaframework.TableCreator.createTables(TableCreator.java:150) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.tools.schemaframework.TableCreator.createTables(TableCreator.java:142) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.tools.schemaframework.SchemaManager.createDefaultTables(SchemaManager.java:1030) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryProvider.generateDefaultTables(EntityManagerFactoryProvider.java:110) [org.eclipse.persistence.jpa-2.7.5.jar:na]
	at org.eclipse.persistence.internal.jpa.EntityManagerSetupImpl.writeDDLToDatabase(EntityManagerSetupImpl.java:4436) [org.eclipse.persistence.jpa-2.7.5.jar:na]
	at org.eclipse.persistence.internal.jpa.EntityManagerSetupImpl.writeMetadataDDLToDatabase(EntityManagerSetupImpl.java:4503) [org.eclipse.persistence.jpa-2.7.5.jar:na]
	at org.eclipse.persistence.internal.jpa.EntityManagerSetupImpl.writeDDL(EntityManagerSetupImpl.java:4402) [org.eclipse.persistence.jpa-2.7.5.jar:na]
	at org.eclipse.persistence.internal.jpa.EntityManagerSetupImpl.writeDDL(EntityManagerSetupImpl.java:4275) [org.eclipse.persistence.jpa-2.7.5.jar:na]
	at org.eclipse.persistence.internal.jpa.EntityManagerSetupImpl.deploy(EntityManagerSetupImpl.java:819) [org.eclipse.persistence.jpa-2.7.5.jar:na]
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryDelegate.getAbstractSession(EntityManagerFactoryDelegate.java:222) [org.eclipse.persistence.jpa-2.7.5.jar:na]
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryDelegate.getDatabaseSession(EntityManagerFactoryDelegate.java:200) [org.eclipse.persistence.jpa-2.7.5.jar:na]
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryImpl.getDatabaseSession(EntityManagerFactoryImpl.java:542) [org.eclipse.persistence.jpa-2.7.5.jar:na]
	at org.eclipse.persistence.jpa.PersistenceProvider.createEntityManagerFactoryImpl(PersistenceProvider.java:153) [org.eclipse.persistence.jpa-2.7.5.jar:na]
	at org.eclipse.persistence.jpa.PersistenceProvider.createEntityManagerFactory(PersistenceProvider.java:191) [org.eclipse.persistence.jpa-2.7.5.jar:na]
	at javax.persistence.Persistence.createEntityManagerFactory(Persistence.java:80) [jakarta.persistence-2.2.3.jar:2.2.3]
	at org.bkvm.cache.MetadataCache.<init>(MetadataCache.java:46) [bkvm-2.0.0-classes.jar:na]
	at org.bkvm.api.listeners.ContextInitializer.contextInitialized(ContextInitializer.java:72) [bkvm-2.0.0-classes.jar:na]
	at org.apache.catalina.core.StandardContext.listenerStart(StandardContext.java:4792) [tomcat-embed-core-8.5.31.jar:8.5.31]
	at org.apache.catalina.core.StandardContext.startInternal(StandardContext.java:5256) [tomcat-embed-core-8.5.31.jar:8.5.31]
	at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:150) [tomcat-embed-core-8.5.31.jar:8.5.31]
	at org.apache.catalina.core.ContainerBase$StartChild.call(ContainerBase.java:1421) [tomcat-embed-core-8.5.31.jar:8.5.31]
	at org.apache.catalina.core.ContainerBase$StartChild.call(ContainerBase.java:1411) [tomcat-embed-core-8.5.31.jar:8.5.31]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_201]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [na:1.8.0_201]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [na:1.8.0_201]
	at java.lang.Thread.run(Thread.java:748) [na:1.8.0_201]
Caused by: org.apache.calcite.runtime.CalciteContextException: From line 1, column 15 to line 1, column 20: Object 'LEDGER' not found
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[na:1.8.0_201]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[na:1.8.0_201]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.8.0_201]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423) ~[na:1.8.0_201]
	at org.apache.calcite.runtime.Resources$ExInstWithCause.ex(Resources.java:457) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.SqlUtil.newContextException(SqlUtil.java:857) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.SqlUtil.newContextException(SqlUtil.java:842) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.SqlValidatorImpl.newValidationError(SqlValidatorImpl.java:5009) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.IdentifierNamespace.resolveImpl(IdentifierNamespace.java:179) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.IdentifierNamespace.validateImpl(IdentifierNamespace.java:184) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.AbstractNamespace.validate(AbstractNamespace.java:84) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.SqlValidatorImpl.validateNamespace(SqlValidatorImpl.java:1089) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.SqlValidatorImpl.validateQuery(SqlValidatorImpl.java:1063) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.SqlValidatorImpl.validateFrom(SqlValidatorImpl.java:3191) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.SqlValidatorImpl.validateFrom(SqlValidatorImpl.java:3173) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.SqlValidatorImpl.validateSelect(SqlValidatorImpl.java:3447) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.SelectNamespace.validateImpl(SelectNamespace.java:60) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.AbstractNamespace.validate(AbstractNamespace.java:84) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.SqlValidatorImpl.validateNamespace(SqlValidatorImpl.java:1089) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.SqlValidatorImpl.validateQuery(SqlValidatorImpl.java:1063) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.SqlSelect.validate(SqlSelect.java:232) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.SqlValidatorImpl.validateScopedExpression(SqlValidatorImpl.java:1038) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.SqlValidatorImpl.validate(SqlValidatorImpl.java:746) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.prepare.PlannerImpl.validate(PlannerImpl.java:215) ~[calcite-core-1.24.0.jar:1.24.0]
	... 59 common frames omitted
Caused by: org.apache.calcite.sql.validate.SqlValidatorException: Object 'LEDGER' not found
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[na:1.8.0_201]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[na:1.8.0_201]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.8.0_201]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423) ~[na:1.8.0_201]
	at org.apache.calcite.runtime.Resources$ExInstWithCause.ex(Resources.java:457) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.runtime.Resources$ExInst.ex(Resources.java:550) ~[calcite-core-1.24.0.jar:1.24.0]
	... 78 common frames omitted

2020-09-09 21:39:42.638  INFO 51244 --- [ost-startStop-1] herddb.core.TableManager                 : loading table ledger, uuid e9e7bcff-bc6e-4a12-9cf5-e8d4ef288e88
2020-09-09 21:39:42.644  INFO 51244 --- [ost-startStop-1] herddb.sql.CalcitePlanner                : Error while parsing 'SELECT 1 FROM LEDGER_METADATA'

org.apache.calcite.tools.ValidationException: org.apache.calcite.runtime.CalciteContextException: From line 1, column 15 to line 1, column 29: Object 'LEDGER_METADATA' not found
	at org.apache.calcite.prepare.PlannerImpl.validate(PlannerImpl.java:217) ~[calcite-core-1.24.0.jar:1.24.0]
	at herddb.sql.CalcitePlanner.runPlanner(CalcitePlanner.java:565) ~[herddb-core-0.19.0.jar:na]
	at herddb.sql.CalcitePlanner.translate(CalcitePlanner.java:331) ~[herddb-core-0.19.0.jar:na]
	at herddb.server.ServerSideConnectionPeer.handleOpenScanner(ServerSideConnectionPeer.java:447) [herddb-core-0.19.0.jar:na]
	at herddb.server.ServerSideConnectionPeer.requestReceived(ServerSideConnectionPeer.java:183) [herddb-core-0.19.0.jar:na]
	at herddb.network.netty.AbstractChannel.processRequest(AbstractChannel.java:96) [herddb-net-0.19.0.jar:na]
	at herddb.network.netty.AbstractChannel.directProcessPdu(AbstractChannel.java:88) [herddb-net-0.19.0.jar:na]
	at herddb.network.netty.LocalVMChannel.sendOneWayMessage(LocalVMChannel.java:63) [herddb-net-0.19.0.jar:na]
	at herddb.network.netty.AbstractChannel.sendRequestWithAsyncReply(AbstractChannel.java:177) [herddb-net-0.19.0.jar:na]
	at herddb.network.Channel.sendMessageWithPduReply(Channel.java:81) [herddb-net-0.19.0.jar:na]
	at herddb.client.RoutedClientSideConnection.executeScan(RoutedClientSideConnection.java:715) [herddb-core-0.19.0.jar:na]
	at herddb.client.HDBConnection.executeScan(HDBConnection.java:345) [herddb-core-0.19.0.jar:na]
	at herddb.jdbc.HerdDBPreparedStatement.executeQuery(HerdDBPreparedStatement.java:81) [herddb-jdbc-0.19.0.jar:na]
	at org.eclipse.persistence.internal.databaseaccess.DatabaseAccessor.executeSelect(DatabaseAccessor.java:1015) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.internal.databaseaccess.DatabaseAccessor.basicExecuteCall(DatabaseAccessor.java:651) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.internal.databaseaccess.DatabaseAccessor.executeCall(DatabaseAccessor.java:567) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.internal.sessions.AbstractSession.basicExecuteCall(AbstractSession.java:2099) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.sessions.server.ServerSession.executeCall(ServerSession.java:603) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.internal.queries.DatasourceCallQueryMechanism.executeCall(DatasourceCallQueryMechanism.java:275) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.internal.queries.DatasourceCallQueryMechanism.executeCall(DatasourceCallQueryMechanism.java:261) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.internal.queries.DatasourceCallQueryMechanism.executeSelectCall(DatasourceCallQueryMechanism.java:332) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.internal.queries.DatasourceCallQueryMechanism.executeSelect(DatasourceCallQueryMechanism.java:314) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.queries.DataReadQuery.executeNonCursor(DataReadQuery.java:199) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.queries.DataReadQuery.executeDatabaseQuery(DataReadQuery.java:154) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.queries.DatabaseQuery.execute(DatabaseQuery.java:911) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.queries.DataReadQuery.execute(DataReadQuery.java:139) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.internal.sessions.AbstractSession.internalExecuteQuery(AbstractSession.java:3356) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.internal.sessions.AbstractSession.executeQuery(AbstractSession.java:1898) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.internal.sessions.AbstractSession.executeQuery(AbstractSession.java:1880) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.internal.sessions.AbstractSession.executeQuery(AbstractSession.java:1830) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.internal.databaseaccess.DatabasePlatform.checkTableExists(DatabasePlatform.java:3775) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.tools.schemaframework.SchemaManager.checkTableExists(SchemaManager.java:467) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.tools.schemaframework.SchemaManager.checkTableExists(SchemaManager.java:480) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.tools.schemaframework.TableCreator.createTables(TableCreator.java:168) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.tools.schemaframework.TableCreator.createTables(TableCreator.java:150) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.tools.schemaframework.TableCreator.createTables(TableCreator.java:142) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.tools.schemaframework.SchemaManager.createDefaultTables(SchemaManager.java:1030) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryProvider.generateDefaultTables(EntityManagerFactoryProvider.java:110) [org.eclipse.persistence.jpa-2.7.5.jar:na]
	at org.eclipse.persistence.internal.jpa.EntityManagerSetupImpl.writeDDLToDatabase(EntityManagerSetupImpl.java:4436) [org.eclipse.persistence.jpa-2.7.5.jar:na]
	at org.eclipse.persistence.internal.jpa.EntityManagerSetupImpl.writeMetadataDDLToDatabase(EntityManagerSetupImpl.java:4503) [org.eclipse.persistence.jpa-2.7.5.jar:na]
	at org.eclipse.persistence.internal.jpa.EntityManagerSetupImpl.writeDDL(EntityManagerSetupImpl.java:4402) [org.eclipse.persistence.jpa-2.7.5.jar:na]
	at org.eclipse.persistence.internal.jpa.EntityManagerSetupImpl.writeDDL(EntityManagerSetupImpl.java:4275) [org.eclipse.persistence.jpa-2.7.5.jar:na]
	at org.eclipse.persistence.internal.jpa.EntityManagerSetupImpl.deploy(EntityManagerSetupImpl.java:819) [org.eclipse.persistence.jpa-2.7.5.jar:na]
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryDelegate.getAbstractSession(EntityManagerFactoryDelegate.java:222) [org.eclipse.persistence.jpa-2.7.5.jar:na]
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryDelegate.getDatabaseSession(EntityManagerFactoryDelegate.java:200) [org.eclipse.persistence.jpa-2.7.5.jar:na]
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryImpl.getDatabaseSession(EntityManagerFactoryImpl.java:542) [org.eclipse.persistence.jpa-2.7.5.jar:na]
	at org.eclipse.persistence.jpa.PersistenceProvider.createEntityManagerFactoryImpl(PersistenceProvider.java:153) [org.eclipse.persistence.jpa-2.7.5.jar:na]
	at org.eclipse.persistence.jpa.PersistenceProvider.createEntityManagerFactory(PersistenceProvider.java:191) [org.eclipse.persistence.jpa-2.7.5.jar:na]
	at javax.persistence.Persistence.createEntityManagerFactory(Persistence.java:80) [jakarta.persistence-2.2.3.jar:2.2.3]
	at org.bkvm.cache.MetadataCache.<init>(MetadataCache.java:46) [bkvm-2.0.0-classes.jar:na]
	at org.bkvm.api.listeners.ContextInitializer.contextInitialized(ContextInitializer.java:72) [bkvm-2.0.0-classes.jar:na]
	at org.apache.catalina.core.StandardContext.listenerStart(StandardContext.java:4792) [tomcat-embed-core-8.5.31.jar:8.5.31]
	at org.apache.catalina.core.StandardContext.startInternal(StandardContext.java:5256) [tomcat-embed-core-8.5.31.jar:8.5.31]
	at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:150) [tomcat-embed-core-8.5.31.jar:8.5.31]
	at org.apache.catalina.core.ContainerBase$StartChild.call(ContainerBase.java:1421) [tomcat-embed-core-8.5.31.jar:8.5.31]
	at org.apache.catalina.core.ContainerBase$StartChild.call(ContainerBase.java:1411) [tomcat-embed-core-8.5.31.jar:8.5.31]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_201]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [na:1.8.0_201]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [na:1.8.0_201]
	at java.lang.Thread.run(Thread.java:748) [na:1.8.0_201]
Caused by: org.apache.calcite.runtime.CalciteContextException: From line 1, column 15 to line 1, column 29: Object 'LEDGER_METADATA' not found
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[na:1.8.0_201]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[na:1.8.0_201]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.8.0_201]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423) ~[na:1.8.0_201]
	at org.apache.calcite.runtime.Resources$ExInstWithCause.ex(Resources.java:457) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.SqlUtil.newContextException(SqlUtil.java:857) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.SqlUtil.newContextException(SqlUtil.java:842) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.SqlValidatorImpl.newValidationError(SqlValidatorImpl.java:5009) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.IdentifierNamespace.resolveImpl(IdentifierNamespace.java:179) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.IdentifierNamespace.validateImpl(IdentifierNamespace.java:184) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.AbstractNamespace.validate(AbstractNamespace.java:84) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.SqlValidatorImpl.validateNamespace(SqlValidatorImpl.java:1089) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.SqlValidatorImpl.validateQuery(SqlValidatorImpl.java:1063) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.SqlValidatorImpl.validateFrom(SqlValidatorImpl.java:3191) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.SqlValidatorImpl.validateFrom(SqlValidatorImpl.java:3173) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.SqlValidatorImpl.validateSelect(SqlValidatorImpl.java:3447) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.SelectNamespace.validateImpl(SelectNamespace.java:60) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.AbstractNamespace.validate(AbstractNamespace.java:84) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.SqlValidatorImpl.validateNamespace(SqlValidatorImpl.java:1089) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.SqlValidatorImpl.validateQuery(SqlValidatorImpl.java:1063) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.SqlSelect.validate(SqlSelect.java:232) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.SqlValidatorImpl.validateScopedExpression(SqlValidatorImpl.java:1038) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.SqlValidatorImpl.validate(SqlValidatorImpl.java:746) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.prepare.PlannerImpl.validate(PlannerImpl.java:215) ~[calcite-core-1.24.0.jar:1.24.0]
	... 59 common frames omitted
Caused by: org.apache.calcite.sql.validate.SqlValidatorException: Object 'LEDGER_METADATA' not found
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[na:1.8.0_201]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[na:1.8.0_201]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.8.0_201]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423) ~[na:1.8.0_201]
	at org.apache.calcite.runtime.Resources$ExInstWithCause.ex(Resources.java:457) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.runtime.Resources$ExInst.ex(Resources.java:550) ~[calcite-core-1.24.0.jar:1.24.0]
	... 78 common frames omitted

2020-09-09 21:39:42.652  INFO 51244 --- [ost-startStop-1] herddb.core.TableManager                 : loading table ledger_metadata, uuid b34ad6c5-6e59-4c32-be7a-e66eb1c1bc9f
2020-09-09 21:39:42.659  INFO 51244 --- [ost-startStop-1] herddb.sql.CalcitePlanner                : Error while parsing 'SELECT 1 FROM LEDGER_BOOKIE'

org.apache.calcite.tools.ValidationException: org.apache.calcite.runtime.CalciteContextException: From line 1, column 15 to line 1, column 27: Object 'LEDGER_BOOKIE' not found
	at org.apache.calcite.prepare.PlannerImpl.validate(PlannerImpl.java:217) ~[calcite-core-1.24.0.jar:1.24.0]
	at herddb.sql.CalcitePlanner.runPlanner(CalcitePlanner.java:565) ~[herddb-core-0.19.0.jar:na]
	at herddb.sql.CalcitePlanner.translate(CalcitePlanner.java:331) ~[herddb-core-0.19.0.jar:na]
	at herddb.server.ServerSideConnectionPeer.handleOpenScanner(ServerSideConnectionPeer.java:447) [herddb-core-0.19.0.jar:na]
	at herddb.server.ServerSideConnectionPeer.requestReceived(ServerSideConnectionPeer.java:183) [herddb-core-0.19.0.jar:na]
	at herddb.network.netty.AbstractChannel.processRequest(AbstractChannel.java:96) [herddb-net-0.19.0.jar:na]
	at herddb.network.netty.AbstractChannel.directProcessPdu(AbstractChannel.java:88) [herddb-net-0.19.0.jar:na]
	at herddb.network.netty.LocalVMChannel.sendOneWayMessage(LocalVMChannel.java:63) [herddb-net-0.19.0.jar:na]
	at herddb.network.netty.AbstractChannel.sendRequestWithAsyncReply(AbstractChannel.java:177) [herddb-net-0.19.0.jar:na]
	at herddb.network.Channel.sendMessageWithPduReply(Channel.java:81) [herddb-net-0.19.0.jar:na]
	at herddb.client.RoutedClientSideConnection.executeScan(RoutedClientSideConnection.java:715) [herddb-core-0.19.0.jar:na]
	at herddb.client.HDBConnection.executeScan(HDBConnection.java:345) [herddb-core-0.19.0.jar:na]
	at herddb.jdbc.HerdDBPreparedStatement.executeQuery(HerdDBPreparedStatement.java:81) [herddb-jdbc-0.19.0.jar:na]
	at org.eclipse.persistence.internal.databaseaccess.DatabaseAccessor.executeSelect(DatabaseAccessor.java:1015) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.internal.databaseaccess.DatabaseAccessor.basicExecuteCall(DatabaseAccessor.java:651) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.internal.databaseaccess.DatabaseAccessor.executeCall(DatabaseAccessor.java:567) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.internal.sessions.AbstractSession.basicExecuteCall(AbstractSession.java:2099) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.sessions.server.ServerSession.executeCall(ServerSession.java:603) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.internal.queries.DatasourceCallQueryMechanism.executeCall(DatasourceCallQueryMechanism.java:275) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.internal.queries.DatasourceCallQueryMechanism.executeCall(DatasourceCallQueryMechanism.java:261) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.internal.queries.DatasourceCallQueryMechanism.executeSelectCall(DatasourceCallQueryMechanism.java:332) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.internal.queries.DatasourceCallQueryMechanism.executeSelect(DatasourceCallQueryMechanism.java:314) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.queries.DataReadQuery.executeNonCursor(DataReadQuery.java:199) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.queries.DataReadQuery.executeDatabaseQuery(DataReadQuery.java:154) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.queries.DatabaseQuery.execute(DatabaseQuery.java:911) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.queries.DataReadQuery.execute(DataReadQuery.java:139) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.internal.sessions.AbstractSession.internalExecuteQuery(AbstractSession.java:3356) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.internal.sessions.AbstractSession.executeQuery(AbstractSession.java:1898) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.internal.sessions.AbstractSession.executeQuery(AbstractSession.java:1880) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.internal.sessions.AbstractSession.executeQuery(AbstractSession.java:1830) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.internal.databaseaccess.DatabasePlatform.checkTableExists(DatabasePlatform.java:3775) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.tools.schemaframework.SchemaManager.checkTableExists(SchemaManager.java:467) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.tools.schemaframework.SchemaManager.checkTableExists(SchemaManager.java:480) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.tools.schemaframework.TableCreator.createTables(TableCreator.java:168) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.tools.schemaframework.TableCreator.createTables(TableCreator.java:150) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.tools.schemaframework.TableCreator.createTables(TableCreator.java:142) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.tools.schemaframework.SchemaManager.createDefaultTables(SchemaManager.java:1030) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryProvider.generateDefaultTables(EntityManagerFactoryProvider.java:110) [org.eclipse.persistence.jpa-2.7.5.jar:na]
	at org.eclipse.persistence.internal.jpa.EntityManagerSetupImpl.writeDDLToDatabase(EntityManagerSetupImpl.java:4436) [org.eclipse.persistence.jpa-2.7.5.jar:na]
	at org.eclipse.persistence.internal.jpa.EntityManagerSetupImpl.writeMetadataDDLToDatabase(EntityManagerSetupImpl.java:4503) [org.eclipse.persistence.jpa-2.7.5.jar:na]
	at org.eclipse.persistence.internal.jpa.EntityManagerSetupImpl.writeDDL(EntityManagerSetupImpl.java:4402) [org.eclipse.persistence.jpa-2.7.5.jar:na]
	at org.eclipse.persistence.internal.jpa.EntityManagerSetupImpl.writeDDL(EntityManagerSetupImpl.java:4275) [org.eclipse.persistence.jpa-2.7.5.jar:na]
	at org.eclipse.persistence.internal.jpa.EntityManagerSetupImpl.deploy(EntityManagerSetupImpl.java:819) [org.eclipse.persistence.jpa-2.7.5.jar:na]
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryDelegate.getAbstractSession(EntityManagerFactoryDelegate.java:222) [org.eclipse.persistence.jpa-2.7.5.jar:na]
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryDelegate.getDatabaseSession(EntityManagerFactoryDelegate.java:200) [org.eclipse.persistence.jpa-2.7.5.jar:na]
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryImpl.getDatabaseSession(EntityManagerFactoryImpl.java:542) [org.eclipse.persistence.jpa-2.7.5.jar:na]
	at org.eclipse.persistence.jpa.PersistenceProvider.createEntityManagerFactoryImpl(PersistenceProvider.java:153) [org.eclipse.persistence.jpa-2.7.5.jar:na]
	at org.eclipse.persistence.jpa.PersistenceProvider.createEntityManagerFactory(PersistenceProvider.java:191) [org.eclipse.persistence.jpa-2.7.5.jar:na]
	at javax.persistence.Persistence.createEntityManagerFactory(Persistence.java:80) [jakarta.persistence-2.2.3.jar:2.2.3]
	at org.bkvm.cache.MetadataCache.<init>(MetadataCache.java:46) [bkvm-2.0.0-classes.jar:na]
	at org.bkvm.api.listeners.ContextInitializer.contextInitialized(ContextInitializer.java:72) [bkvm-2.0.0-classes.jar:na]
	at org.apache.catalina.core.StandardContext.listenerStart(StandardContext.java:4792) [tomcat-embed-core-8.5.31.jar:8.5.31]
	at org.apache.catalina.core.StandardContext.startInternal(StandardContext.java:5256) [tomcat-embed-core-8.5.31.jar:8.5.31]
	at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:150) [tomcat-embed-core-8.5.31.jar:8.5.31]
	at org.apache.catalina.core.ContainerBase$StartChild.call(ContainerBase.java:1421) [tomcat-embed-core-8.5.31.jar:8.5.31]
	at org.apache.catalina.core.ContainerBase$StartChild.call(ContainerBase.java:1411) [tomcat-embed-core-8.5.31.jar:8.5.31]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_201]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [na:1.8.0_201]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [na:1.8.0_201]
	at java.lang.Thread.run(Thread.java:748) [na:1.8.0_201]
Caused by: org.apache.calcite.runtime.CalciteContextException: From line 1, column 15 to line 1, column 27: Object 'LEDGER_BOOKIE' not found
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[na:1.8.0_201]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[na:1.8.0_201]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.8.0_201]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423) ~[na:1.8.0_201]
	at org.apache.calcite.runtime.Resources$ExInstWithCause.ex(Resources.java:457) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.SqlUtil.newContextException(SqlUtil.java:857) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.SqlUtil.newContextException(SqlUtil.java:842) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.SqlValidatorImpl.newValidationError(SqlValidatorImpl.java:5009) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.IdentifierNamespace.resolveImpl(IdentifierNamespace.java:179) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.IdentifierNamespace.validateImpl(IdentifierNamespace.java:184) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.AbstractNamespace.validate(AbstractNamespace.java:84) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.SqlValidatorImpl.validateNamespace(SqlValidatorImpl.java:1089) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.SqlValidatorImpl.validateQuery(SqlValidatorImpl.java:1063) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.SqlValidatorImpl.validateFrom(SqlValidatorImpl.java:3191) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.SqlValidatorImpl.validateFrom(SqlValidatorImpl.java:3173) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.SqlValidatorImpl.validateSelect(SqlValidatorImpl.java:3447) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.SelectNamespace.validateImpl(SelectNamespace.java:60) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.AbstractNamespace.validate(AbstractNamespace.java:84) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.SqlValidatorImpl.validateNamespace(SqlValidatorImpl.java:1089) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.SqlValidatorImpl.validateQuery(SqlValidatorImpl.java:1063) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.SqlSelect.validate(SqlSelect.java:232) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.SqlValidatorImpl.validateScopedExpression(SqlValidatorImpl.java:1038) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.SqlValidatorImpl.validate(SqlValidatorImpl.java:746) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.prepare.PlannerImpl.validate(PlannerImpl.java:215) ~[calcite-core-1.24.0.jar:1.24.0]
	... 59 common frames omitted
Caused by: org.apache.calcite.sql.validate.SqlValidatorException: Object 'LEDGER_BOOKIE' not found
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[na:1.8.0_201]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[na:1.8.0_201]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.8.0_201]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423) ~[na:1.8.0_201]
	at org.apache.calcite.runtime.Resources$ExInstWithCause.ex(Resources.java:457) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.runtime.Resources$ExInst.ex(Resources.java:550) ~[calcite-core-1.24.0.jar:1.24.0]
	... 78 common frames omitted

2020-09-09 21:39:42.665  INFO 51244 --- [ost-startStop-1] herddb.core.TableManager                 : loading table ledger_bookie, uuid bce88e9b-9a6a-4691-ad95-56d64b364e36
2020-09-09 21:39:42.672  INFO 51244 --- [ost-startStop-1] herddb.sql.CalcitePlanner                : Error while parsing 'SELECT 1 FROM BOOKIE'

org.apache.calcite.tools.ValidationException: org.apache.calcite.runtime.CalciteContextException: From line 1, column 15 to line 1, column 20: Object 'BOOKIE' not found
	at org.apache.calcite.prepare.PlannerImpl.validate(PlannerImpl.java:217) ~[calcite-core-1.24.0.jar:1.24.0]
	at herddb.sql.CalcitePlanner.runPlanner(CalcitePlanner.java:565) ~[herddb-core-0.19.0.jar:na]
	at herddb.sql.CalcitePlanner.translate(CalcitePlanner.java:331) ~[herddb-core-0.19.0.jar:na]
	at herddb.server.ServerSideConnectionPeer.handleOpenScanner(ServerSideConnectionPeer.java:447) [herddb-core-0.19.0.jar:na]
	at herddb.server.ServerSideConnectionPeer.requestReceived(ServerSideConnectionPeer.java:183) [herddb-core-0.19.0.jar:na]
	at herddb.network.netty.AbstractChannel.processRequest(AbstractChannel.java:96) [herddb-net-0.19.0.jar:na]
	at herddb.network.netty.AbstractChannel.directProcessPdu(AbstractChannel.java:88) [herddb-net-0.19.0.jar:na]
	at herddb.network.netty.LocalVMChannel.sendOneWayMessage(LocalVMChannel.java:63) [herddb-net-0.19.0.jar:na]
	at herddb.network.netty.AbstractChannel.sendRequestWithAsyncReply(AbstractChannel.java:177) [herddb-net-0.19.0.jar:na]
	at herddb.network.Channel.sendMessageWithPduReply(Channel.java:81) [herddb-net-0.19.0.jar:na]
	at herddb.client.RoutedClientSideConnection.executeScan(RoutedClientSideConnection.java:715) [herddb-core-0.19.0.jar:na]
	at herddb.client.HDBConnection.executeScan(HDBConnection.java:345) [herddb-core-0.19.0.jar:na]
	at herddb.jdbc.HerdDBPreparedStatement.executeQuery(HerdDBPreparedStatement.java:81) [herddb-jdbc-0.19.0.jar:na]
	at org.eclipse.persistence.internal.databaseaccess.DatabaseAccessor.executeSelect(DatabaseAccessor.java:1015) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.internal.databaseaccess.DatabaseAccessor.basicExecuteCall(DatabaseAccessor.java:651) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.internal.databaseaccess.DatabaseAccessor.executeCall(DatabaseAccessor.java:567) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.internal.sessions.AbstractSession.basicExecuteCall(AbstractSession.java:2099) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.sessions.server.ServerSession.executeCall(ServerSession.java:603) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.internal.queries.DatasourceCallQueryMechanism.executeCall(DatasourceCallQueryMechanism.java:275) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.internal.queries.DatasourceCallQueryMechanism.executeCall(DatasourceCallQueryMechanism.java:261) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.internal.queries.DatasourceCallQueryMechanism.executeSelectCall(DatasourceCallQueryMechanism.java:332) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.internal.queries.DatasourceCallQueryMechanism.executeSelect(DatasourceCallQueryMechanism.java:314) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.queries.DataReadQuery.executeNonCursor(DataReadQuery.java:199) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.queries.DataReadQuery.executeDatabaseQuery(DataReadQuery.java:154) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.queries.DatabaseQuery.execute(DatabaseQuery.java:911) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.queries.DataReadQuery.execute(DataReadQuery.java:139) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.internal.sessions.AbstractSession.internalExecuteQuery(AbstractSession.java:3356) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.internal.sessions.AbstractSession.executeQuery(AbstractSession.java:1898) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.internal.sessions.AbstractSession.executeQuery(AbstractSession.java:1880) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.internal.sessions.AbstractSession.executeQuery(AbstractSession.java:1830) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.internal.databaseaccess.DatabasePlatform.checkTableExists(DatabasePlatform.java:3775) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.tools.schemaframework.SchemaManager.checkTableExists(SchemaManager.java:467) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.tools.schemaframework.SchemaManager.checkTableExists(SchemaManager.java:480) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.tools.schemaframework.TableCreator.createTables(TableCreator.java:168) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.tools.schemaframework.TableCreator.createTables(TableCreator.java:150) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.tools.schemaframework.TableCreator.createTables(TableCreator.java:142) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.tools.schemaframework.SchemaManager.createDefaultTables(SchemaManager.java:1030) [org.eclipse.persistence.core-2.7.5.jar:na]
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryProvider.generateDefaultTables(EntityManagerFactoryProvider.java:110) [org.eclipse.persistence.jpa-2.7.5.jar:na]
	at org.eclipse.persistence.internal.jpa.EntityManagerSetupImpl.writeDDLToDatabase(EntityManagerSetupImpl.java:4436) [org.eclipse.persistence.jpa-2.7.5.jar:na]
	at org.eclipse.persistence.internal.jpa.EntityManagerSetupImpl.writeMetadataDDLToDatabase(EntityManagerSetupImpl.java:4503) [org.eclipse.persistence.jpa-2.7.5.jar:na]
	at org.eclipse.persistence.internal.jpa.EntityManagerSetupImpl.writeDDL(EntityManagerSetupImpl.java:4402) [org.eclipse.persistence.jpa-2.7.5.jar:na]
	at org.eclipse.persistence.internal.jpa.EntityManagerSetupImpl.writeDDL(EntityManagerSetupImpl.java:4275) [org.eclipse.persistence.jpa-2.7.5.jar:na]
	at org.eclipse.persistence.internal.jpa.EntityManagerSetupImpl.deploy(EntityManagerSetupImpl.java:819) [org.eclipse.persistence.jpa-2.7.5.jar:na]
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryDelegate.getAbstractSession(EntityManagerFactoryDelegate.java:222) [org.eclipse.persistence.jpa-2.7.5.jar:na]
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryDelegate.getDatabaseSession(EntityManagerFactoryDelegate.java:200) [org.eclipse.persistence.jpa-2.7.5.jar:na]
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryImpl.getDatabaseSession(EntityManagerFactoryImpl.java:542) [org.eclipse.persistence.jpa-2.7.5.jar:na]
	at org.eclipse.persistence.jpa.PersistenceProvider.createEntityManagerFactoryImpl(PersistenceProvider.java:153) [org.eclipse.persistence.jpa-2.7.5.jar:na]
	at org.eclipse.persistence.jpa.PersistenceProvider.createEntityManagerFactory(PersistenceProvider.java:191) [org.eclipse.persistence.jpa-2.7.5.jar:na]
	at javax.persistence.Persistence.createEntityManagerFactory(Persistence.java:80) [jakarta.persistence-2.2.3.jar:2.2.3]
	at org.bkvm.cache.MetadataCache.<init>(MetadataCache.java:46) [bkvm-2.0.0-classes.jar:na]
	at org.bkvm.api.listeners.ContextInitializer.contextInitialized(ContextInitializer.java:72) [bkvm-2.0.0-classes.jar:na]
	at org.apache.catalina.core.StandardContext.listenerStart(StandardContext.java:4792) [tomcat-embed-core-8.5.31.jar:8.5.31]
	at org.apache.catalina.core.StandardContext.startInternal(StandardContext.java:5256) [tomcat-embed-core-8.5.31.jar:8.5.31]
	at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:150) [tomcat-embed-core-8.5.31.jar:8.5.31]
	at org.apache.catalina.core.ContainerBase$StartChild.call(ContainerBase.java:1421) [tomcat-embed-core-8.5.31.jar:8.5.31]
	at org.apache.catalina.core.ContainerBase$StartChild.call(ContainerBase.java:1411) [tomcat-embed-core-8.5.31.jar:8.5.31]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_201]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [na:1.8.0_201]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [na:1.8.0_201]
	at java.lang.Thread.run(Thread.java:748) [na:1.8.0_201]
Caused by: org.apache.calcite.runtime.CalciteContextException: From line 1, column 15 to line 1, column 20: Object 'BOOKIE' not found
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[na:1.8.0_201]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[na:1.8.0_201]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.8.0_201]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423) ~[na:1.8.0_201]
	at org.apache.calcite.runtime.Resources$ExInstWithCause.ex(Resources.java:457) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.SqlUtil.newContextException(SqlUtil.java:857) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.SqlUtil.newContextException(SqlUtil.java:842) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.SqlValidatorImpl.newValidationError(SqlValidatorImpl.java:5009) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.IdentifierNamespace.resolveImpl(IdentifierNamespace.java:179) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.IdentifierNamespace.validateImpl(IdentifierNamespace.java:184) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.AbstractNamespace.validate(AbstractNamespace.java:84) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.SqlValidatorImpl.validateNamespace(SqlValidatorImpl.java:1089) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.SqlValidatorImpl.validateQuery(SqlValidatorImpl.java:1063) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.SqlValidatorImpl.validateFrom(SqlValidatorImpl.java:3191) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.SqlValidatorImpl.validateFrom(SqlValidatorImpl.java:3173) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.SqlValidatorImpl.validateSelect(SqlValidatorImpl.java:3447) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.SelectNamespace.validateImpl(SelectNamespace.java:60) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.AbstractNamespace.validate(AbstractNamespace.java:84) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.SqlValidatorImpl.validateNamespace(SqlValidatorImpl.java:1089) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.SqlValidatorImpl.validateQuery(SqlValidatorImpl.java:1063) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.SqlSelect.validate(SqlSelect.java:232) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.SqlValidatorImpl.validateScopedExpression(SqlValidatorImpl.java:1038) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.sql.validate.SqlValidatorImpl.validate(SqlValidatorImpl.java:746) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.prepare.PlannerImpl.validate(PlannerImpl.java:215) ~[calcite-core-1.24.0.jar:1.24.0]
	... 59 common frames omitted
Caused by: org.apache.calcite.sql.validate.SqlValidatorException: Object 'BOOKIE' not found
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[na:1.8.0_201]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[na:1.8.0_201]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.8.0_201]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423) ~[na:1.8.0_201]
	at org.apache.calcite.runtime.Resources$ExInstWithCause.ex(Resources.java:457) ~[calcite-core-1.24.0.jar:1.24.0]
	at org.apache.calcite.runtime.Resources$ExInst.ex(Resources.java:550) ~[calcite-core-1.24.0.jar:1.24.0]
	... 78 common frames omitted
```

--- a/src/main/resources/bkvm.conf
+++ b/src/main/resources/bkvm.conf
@@ -19,6 +19,10 @@ bkvm.enabled=false
 # Default value zk+null://127.0.0.1:2181/ledgers works for Pulsar Standalone
 metadataServiceUri=zk+null://127.0.0.1:2181/ledgers
 
+# Refresh BK metadata at boot.
+# BK metadata are not scanned automatically in BKVM, you have to request it from the UI
+metdata.refreshAtBoot=true
+
 # HerdDB database connection, not to be changed if you are running embedded HerdDB in Pulsar Manager
 # If you are using PostGRE SQL you will have to change this configuration
 # We want to use the HerdDB database started by PulsarManager itself, by default BKVM wants to start its one database

